docs/vendor.md

```diff
@@ -0,0 +1,87 @@
+# Vendor Image Requirements
+
+A Vendor Image is a docker container that can be used with KNE to emulate a
+vendor's devices.
+
+Without vendor images KNE is just an empty virtual machine rack that does
+nothing.  Vendor supplied images are what the user of KNE sees and is interested
+in.  This document describes the requirements and expectations of vendor images
+to be used with KNE.
+
+This document describes the image requirements and expectations in order to be
+considered *KNE Qualified*.
```

Comment:
what does KNE qualified mean in this case? Will there be an indicator anywhere that the image is KNE qualified, as in, is this the requirements for us to merge a new node impl? Or do mark some of the node impls as "qualified"?